packages/storybook/stories/components/form-field/form-field.stories.ts

```diff
@@ -225,19 +225,297 @@ export const Tooltip: StoryObj = {
   `,
 };
 
+export const Checkbox: StoryObj = {
+  tags: ['isHidden'],
+  render: () => html`
+<ods-form-field>
+  <span class="form-field__checkbox__field">
+    <ods-checkbox input-id="input-id-example" name="name"></ods-checkbox>
+    <label for="input-id-example">
+      <ods-text class="form-field__checkbox__label" slot="label" preset="label">
```

Comment:
no need for `ods-text` here just use native `label` otherwise you'll end up with two label tags